packages/auto-install/test/npm.js

```diff
@@ -17,7 +17,7 @@ const pkgFile = join(cwd, 'package.json');
 process.chdir(cwd);
 
 test('invalid manager', (t) => {
-  t.timeout(30000);
+  t.timeout(50000);
```

Comment:
I kept seeing random timeouts on these tests, this seems to help a little. Sometimes on Windows, npm seems to be really slow..